components/namespace-claim-cleaner/base/rbac.yaml

```diff
@@ -0,0 +1,38 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: namespace-claim-cleaner
+  namespace: namespace-claim-cleaner
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: namespace-claim-cleaner
+rules:
+  - apiGroups:
+      - toolchain.dev.openshift.com
+    resources:
+      - namespace-claim
+    verbs:
+      - list
+      - delete
```

Comment:
This isn't a valid resource nor should this role/SA interact with anything related to the toolchain operators. Please remove this.